components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/bean/APIResponseFaultCountValue.java

```diff
@@ -0,0 +1,49 @@
+/*
+* Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+*/
+package org.wso2.carbon.apimgt.usage.client.bean;
+
+import java.util.List;
+
+public class APIResponseFaultCountValue {
+    private int totalFaultCount;
+    private List<String> api_version_apiPublisher_context_facet;
```

Comment:
Hi Abimaran,

APIResponseFaultCountValue is a bean class and used to parse to java object from Json response sent from the DAS rest API.

In the Response it has attribute "api_version_apiPublisher_context_facet". That's why i have used the variable name like this. So do we have any alternative for this?

Thanks.